docs/technical_development_guide/source/changelog/changelog_dev.rst

```diff
@@ -7,7 +7,24 @@ Development Changelog
     :version: v3.5.0
     :released: pending
 
-    
+
+    .. change::
+        :tags: workflow, feature, outputs
+        :pullreq: 1368
+
+        **Date**: 2025-04-30
+
+        Title:
+        Latest OS-HPXML
+
+        Description:
+        Introduce new argument for including detailed zone conditions timeseries outputs (i.e., temperatures, humidities).
```

Comment:
Should we add something about EV charging/discharging during unavailable periods? Anything else?

Reply:
I added something about EV charging/discharging during unavailable periods.